### ISSUE TITLE
Let's coordinate

### ISSUE THREAD

**Issue by smiklin** [liu431/DescrStats#1]
@liu431 @hannamn @SixueLiuMACSS

Hello group 5! I'm happy to work on this remotely, but we can set a time when we're all working on it and in touch with each other.

**Comment by liu431** [comment]
Great!

**Comment by SixueLiuMACSS** [comment]
Will you guys be at today's lab?

Sixue
Sixue Liu
Master of Arts in Computational Social Science
Social Science Divisions
University of Chicago

**Comment by liu431** [comment]
I am available on Friday.  How about 1:30 p.m at Saieh?

**Comment by smiklin** [comment]
1:30 pm tomorrow works for me. I should be able to make it to Saieh, but I think it's not a problem if we don't all make it there in person

**Comment by hannamn** [comment]
I can also be there. Should we meet on the tables right outside the class?

**Comment by SixueLiuMACSS** [comment]
Sorry, I may not okay at 1:30pm. May be 3 or 4pm? Does it sound good to your guys?

**Comment by smiklin** [comment]
I could do 3 pm, but need to leave around 4

**Comment by SixueLiuMACSS** [comment]
sounds great, 3pm works for me!

**Comment by liu431** [comment]
3 pm works for me

**Comment by hannamn** [comment]
works for me too!

**Comment by SixueLiuMACSS** [comment]
Hey! You can find me on the tables outside the classroom.  We will meet here, right?

**Comment by smiklin** [comment]
If we are still meeting?
We could go through it once more together with the proper starting point and double check all the comments and see what's happening with those. I could definitely use the practice...

**Comment by smiklin** [comment]
@SixueLiuMACSS are you there? Are we meeting?

**Comment by hannamn** [comment]
I’m on my way to campus, but can easily make my way somewhere else if we’re good and don’t need to meet

**Comment by smiklin** [comment]
Let's just meet (in person everyone!) and touch base

**Comment by SixueLiuMACSS** [comment]
@smiklin Yeah! I am outside our perspective classroom! You can definitely find me here!
